app/assets/stylesheets/style.css.scss

```diff
@@ -181,6 +184,10 @@ a {
   }
 }
 
+.title {
+  padding-top: 100px;
```

Comment:
Also, please make the menu toggle-able (and hide it by default) using mobi.scss.